### PR TITLE
Post: Cactusforce 2k24

### PR DESCRIPTION
**Pull request by scottbcovert**
@ndrewr @martinglauber @chivalry Here is the first draft of the blog post on Cactusforce 2k24. I included everyone as an author in case folks would like to add a paragraph or two on their favorite talks from the conference 👍

### CODE REVIEW

File: _drafts/2024-03-15-cactusforce-2k24.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+slug:
+---
+
+[Andrew Chen](https://www.linkedin.com/in/ndrewr/), [Scott Covert](https://www.linkedin.com/in/scottbcovert/), [Martin Glauber](https://www.linkedin.com/in/martin-glauber-583b143a/), and [Chuck Ross](https://www.linkedin.com/in/chivalry/)
+
+<br/>
+
+### Return to the Copper State
+
+Tython kicked off 2024 by reconvening in Arizona for the annual Cactusforce conference held for Salesforce developers and architects. This year we met in Scottsdale and we were able to have all four members meet in person for the first time!
```
Comment (ndrewr) [inline _drafts/2024-03-15-cactusforce-2k24.md:17]: How about split up the two ideas

> This year we met in Scottsdale and we were able to have  and we were able to have all four members meet in person for the first time!

something like

>Whereas last year the event was held in downtown Phoenix, this year's action took place in nearby Scottsdale. Even more monumental for the team, all four members were able to meet in person for the first time!

File: _drafts/2024-03-15-cactusforce-2k24.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+### Down the Rabbit Hole
+
+The content presented at Cactusforce is intentionally technical and developer-oriented. I had the pleasure this year of leading my own two-part session covering the tradeoffs of different OAuth flows available to ISVs that have loopback integrations embedded in their application to call Salesforce APIs from Apex.
+
+![OAuth Orchestration](/images/2024-03-15-cactusforce-2k24-presentation.jpg)
+
+This topic deserves a post or two of its own so stay tuned for future write-ups on the subject. In the meantime, [you can watch the session recording](https://vimeo.com/showcase/10948502/video/914579193)!
+
+[David Schach](https://www.linkedin.com/in/davidschach/) and [Chuck Liddell](https://www.linkedin.com/in/chuckliddell/) were panelists at the session (thanks again, guys!) and Chuck also had a session of his own on designing data facades for deeply-nested data integrations that was very interesting for anyone who has experienced the joy of parsing JSON in Apex. Chuck and his team built out a solution to the problem of a never ending series of statements that cast results to `Map<String,Object>` that we see too often in this scenario.
```
Comment (ndrewr) [inline _drafts/2024-03-15-cactusforce-2k24.md:31]: >David Schach and Chuck Liddell were panelists at the session (thanks again, guys!) and Chuck also had a session of his own 

can split in twine

>David Schach and Chuck Liddell were panelists at the session (thanks again, guys!). Chuck also had a session of his own ...

File: _drafts/2024-03-15-cactusforce-2k24.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+### Getting Comfortable with AI
+
+We've all noticed the rise of AI technologies in the last few years, and [Karthik Nallajalla](https://www.linkedin.com/in/karthik-nallajalla-9200087/) showed us how [GitHub Copilot](https://github.com/features/copilot/) can integrate with Salesforce's favorite IDE, [VS Code](https://code.visualstudio.com), to more quickly build apps using the vast open source software Copilot has access to. While there's understandable concern over AI and it's implementations, there's no escaping it in our field. Salesforce themselves just finished hosting a heavily AI themed [TrailblzerDX](https://www.salesforce.com/trailblazerdx/) (more on that in a future post).
```
Comment (ndrewr) [inline _drafts/2024-03-15-cactusforce-2k24.md:35]: "TrailblazerDX" typo

Comment (ndrewr) [inline _drafts/2024-03-15-cactusforce-2k24.md:35]: Clicking on the "Karthik" link shows me this message

```
An exact match for karthik-nallajalla-9200087 could not be found.
```

File: _drafts/2024-03-15-cactusforce-2k24.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+One example Karthik highlighted was the ability for Copilot to write unit tests, and a helpful audience member brought up the point that such generated code is only really testing what the code already does, rather than what it should do. The major theme of using AI to generate code is that it's a first draft that must be checked, just as if you're reviewing the code of a junior developer. It'll be fun and interesting to experiment with Copilot and Salesforce's own [Einstein for Developers](https://developer.salesforce.com/tools/vscode/en/einstein/einstein-overview).
```
Comment (ndrewr) [inline _drafts/2024-03-15-cactusforce-2k24.md:37]: > It’ll be fun and interesting to experiment with Copilot and Salesforce’s own Einstein for Developers.

tweak

> It’ll be fun and interesting to experiment with both Copilot as well as Salesforce’s own Einstein for Developers.

**Review by ndrewr — APPROVED** [review]
Thanks for hashing this out, I think your sections are about good to go once the rest of the team has added their input. I made some minor tweak suggestions that could go either way, and I noticed one typo and one potentially broken link.

One suggestion for formatting would be to insert a newline (whitespace) between the major sections led off with a `heading`. 
As well as a newline above the "call to action" line.

File: _drafts/2024-03-15-cactusforce-2k24.md (hunk 3 above)
Comment (chivalry) [reply _drafts/2024-03-15-cactusforce-2k24.md:35]: Strange. Takes me to his linked in page, just copied and pasted it into Safari. I'll do some checking.

**Comment by scottbcovert** [comment]
Great suggestions all around, I think I've incorporated everyone' feedback so going to merge so this will publish tomorrow AM 👍

**Pull request merged**